godot-core/src/builtin/array.rs

```diff
@@ -135,10 +135,23 @@ impl<T: GodotType> Array<T> {
     /// Note: The sorting algorithm used is not
     /// [stable](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability). This means that values
     /// considered equal may have their order changed when using `sort_unstable`.
+    #[doc(alias = "sort")]
     pub fn sort_unstable(&mut self) {
         self.as_inner().sort();
     }
 
+    /// Sorts the array.
+    ///
+    /// Uses the provided `Callable` to determine ordering.
+    ///
+    /// Note: The sorting algorithm used is not
+    /// [stable](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability). This means that values
+    /// considered equal may have their order changed when using `sort_unstable`.
```

Comment:
You can use the [120-145 available width](https://godot-rust.github.io/book/contribute/conventions.html#formatting):

```suggestion
    /// Note: the sorting algorithm used is not [stable](https://en.wikipedia.org/wiki/Sorting_algorithm#Stability).
    /// This means that values considered equal may have their order changed when using `sort_unstable`.
```